src/Microsoft.SqlTools.ServiceLayer/Localization/sr.resx

```diff
@@ -1571,6 +1571,16 @@
     <value>An XEvent session named {0} already exists</value>
     <comment>.
  Parameters: 0 - sessionName (String) </comment>
+  </data>
+  <data name="SessionMissingDetails" xml:space="preserve">
+    <value>Unable to start streaming session {0} due to missing session details.</value>
+    <comment>.

```

Comment:
Do you know why the comments are on a separate line like this? Seems like that's existing behavior but it looks pretty odd...is that something we can fix?

Reply:
This is auto generated Charles, I'm not sure why this is the case, but I'll change it manually for this PR for now.

Reply:
It's generated by the String Resource Tool included, so I suppose this is the expected result.